eclipse-theia/theia
# GH-7521: Enabled `Select All` in the `Output`

#### What it does

It enables the `Select All` functionality in the `Output` widget. Enabled only if the `Output` widget is the active widget in the shell. It should work with <kbd>Ctrl/Cmd</kbd>+<kbd>A</kbd> keybindings by default. Closes #7521.

#### How to test

 - Make you have some input for the `Output` widget. You can use the `API Sample: Log to Output` command.
 - Activate the widget and try to select all with the keybinding.
 - Make sure the `Output` widget is visible but not active
    - perform a `Select All` via the keyboard whilst the editor is active,
    - perform a `Select All` via the keyboard whilst the SIW widget is active and `input` has the focus.
 - Verify the same in electron.
 
#### Review checklist


#### Reminder for reviewers

- as a reviewer, I agree to behave in accordance with [the review guidelines](https://github.com/theia-ide/theia/blob/master/doc/pull-requests.md#reviewing)
